neuroquery/datasets.py

```diff
@@ -72,12 +73,19 @@ def fetch_peak_coordinates(data_dir=None):
         return str(out_file)
     print("Downloading coordinates")
     resp = requests.get(
-        "https://raw.githubusercontent.com/neuroquery/neuroquery_data/"
-        "master/training_data/coordinates.csv"
+        "https://raw.githubusercontent.com/neuroquery/neuroquery_data/master/data/data-neuroquery_version-1_coordinates.tsv.gz"
     )
     resp.raise_for_status()
+    content = zlib.decompress(resp.content, 15 + 32)
```

Comment:
if it is easy to explain, could you add a short comment explaining the "15 + 32"? otherwise I guess we can simply check the zlib documentation